client/components/Promotion.jsx

```diff
@@ -9,15 +9,37 @@ border-radius: 2px;
 width: 95%;
 margin: 0 auto;
 margin-bottom: 2px;
-display: inline-block; 
+display: inline-block;
+border: none;
 `;
+
+const Span = styled.span`
+  color: #d0d5e1;
+  font-size: .95em;
+  font-weight: bold;
+  text-transform: uppercase;
+  text-shadow: 0 2px 0 rgba(63,68,80,0.35);
+`;
+
+const SpanD = styled.span`
+color: #AEFF14;
+font-size: .95em;
+`;
+
+const SpanO = styled.span `
+  color: orange;
+  font-size: .95em;
+`;
+
```

Comment:
I recommend giving these more descriptive names eventually

Reply:
Will go through, audit and rename carefully. Ty Rane